### ISSUE TITLE
Windows support code should use implementer, not implements

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8291]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8291|
|Type|enhancement|
|Created|2016-04-14 11:16:02Z|
|Branch|https://github.com/twisted/twisted/tree/implementer-windows-8291|

Part of [#8025](https://github.com/twisted/twisted/issues/8025), in prep for Python 3 porting

<details><summary>Searchable metadata</summary>

```
trac-id__8291 8291
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_implementer_windows_8291 branches/implementer-windows-8291
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1460632562599273 1460632562599273
changetime__1460683612323177 1460683612323177
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [47228]) Branching to implementer-windows-8291.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

I'm happy with this. Please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set owner to @hawkowl|
|-|-|

all good. please merge. thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

(In [47237]) Merge implementer-windows-8291: Change uses of implements to implementer in Windows support code

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8291](https://github.com/twisted/twisted/issues/8291)
